migrations/educandu-2023-01-13-02-remove-midi-from-ear-training-plugin.js

```diff
@@ -0,0 +1,43 @@
+export default class Educandu_2023_01_13_02_remove_midi_from_ear_training_plugin {
+
+  constructor(db) {
+    this.db = db;
+  }
+
+  async processCollection(collectionName) {
+    const toUpdate = await this.db.collection(collectionName).find({ 'sections.type': 'ear-training' }).toArray();
+    let updateCount = 0;
+
+    for (const doc of toUpdate) {
+      let docWasUpdated = false;
+      for (const section of doc.sections) {
+
+        if (section.type === 'ear-training' && section.content) {
+          section.content.tests.forEach(test => {
+            delete test.sound.useMidi;
```

Comment:
Just as a note, `updateMany` can be used in this case to, with:
`        $unset: {
              'sections.$[sectionElement].content.tests.$[].useMidi': null
          }
`